vhd/release-notes/aks-windows/2019-datacenter-core-smalldisk-17763.1217.200603.txt

```diff
@@ -0,0 +1,120 @@
+c:\akse-cache\win-k8s\v1.15.11-1int.zip                                          FD31594C5D6C3C0EDD97269FB09630EC751353894166521F919C48DB3F58ADD9  99122881
+c:\akse-cache\win-k8s\v1.15.11-azs-1int.zip                                      10576CBA08115EC246CAE44F6AD52ECB26EC88DBBF77A7571F79AD769B5E3B31  99130288
+c:\akse-cache\win-k8s\v1.15.12-1int.zip                                          18E0124DEB357EE6E599DD583AD0B74E3DFBEF41C70E0E79EB1846C5839E5116  99183450
+c:\akse-cache\win-k8s\v1.15.12-azs-1int.zip                                      BC930FFAD50B823B0D15C3DBF21CC2D85CF9C84873C9852BA2F87968D298ECA8  99182019
+c:\akse-cache\win-k8s\v1.16.10-1int.zip                                          E55A3FA96344AB221C71A8760E21590284908A0215DE18D66B7E94D027004E50  98024468
+c:\akse-cache\win-k8s\v1.16.8-1int.zip                                           EC79CB805E21DD0BC07D2E67FDB34A97D53DB104A7F0D3FD4016BD61FA2481BA  97976592
+c:\akse-cache\win-k8s\v1.16.8-azs-1int.zip                                       B92C377A2F7D1B339C92AC4ABAAA108CC96FBA608EFC3464105490BC3B37D558  97973074
+c:\akse-cache\win-k8s\v1.16.9-1int.zip                                           900606A4B3BCD54C8F47874AB427F5187F76A3318DBA96459AE049244823FAC4  97998180
+c:\akse-cache\win-k8s\v1.16.9-azs-1int.zip                                       6FE2C265DEAEFEADE1B3894EAA6550A65E3BB62B2CECEB2FF8D948BD7300273C  98006329
+c:\akse-cache\win-k8s\v1.17.4-1int.zip                                           E47CCA166FFBC63853350DD834FDA436D8737F9BD2188FEF0A18C8CD64450476  98265153
+c:\akse-cache\win-k8s\v1.17.4-azs-1int.zip                                       7A8CECE114E5BF28D70EFEB8183CA037E0B32AF12FE83845F278E1C03C258FAA  98264539
+c:\akse-cache\win-k8s\v1.17.5-1int.zip                                           BD6829690C6699CDCCC40CAF347080DD5935D36A6C5398F22C42D890E920C100  57341983
+c:\akse-cache\win-k8s\v1.17.5-azs-1int.zip                                       3CDA19E607233B8528CAEE7BC0F54851E11FDA2C5FDD55AB8832E56EF46D6168  98279104
+c:\akse-cache\win-k8s\v1.17.6-1int.zip                                           C0E7975B8DFEB877D8A567FE2A47BB85F7661E4300B5189B44F1EE0E290C4DA0  57391554
+c:\akse-cache\win-k8s\v1.18.1-1int.zip                                           6E3A987C8A28ED37D16F0DF006AED3C5D68E4CEE5B09AF5F60199ADDFD23D240  57969682
+c:\akse-cache\win-k8s\v1.18.2-1int.zip                                           6F326FF16F063F7DC3A81CB09715DE5EC3674164E870D4E978C3D915D80076E7  57971836
+c:\akse-cache\win-k8s\v1.18.3-1int.zip                                           EE26BD5FE719E87845078590CDA54B93B27D44C1FE840861E1139648B29B9E5F  58020535
+c:\akse-cache\win-k8s\v1.19.0-beta.0-1int.zip                                    D99EFC6998405F3F8CD3D3AD4B4D57D837133BB07725E966D659DE2801F63F83  60019442
+c:\akse-cache\win-vnet-cni\azure-vnet-cni-singletenancy-windows-amd64-v1.1.2.zip 60F21CAD6439446FCBD1D9A634E7D739D3BF589D17D7D0EAF4A90A63B544466B  23792148

```

Comment:
Fyi in testing with virtual kubelet we found a bug in 1.1.2 with virtual kubelet. We know the last one that past was 1.0.33 and don't know if 1.1.1|0 had it. So in the aks linx vhd we actually kept 1.0.33 and dropped 1.1.1|2. Now the chances of 1.1.3 having to roll back and somone having windows and virtual kubelet are probably pretty low so this is an fyi rather than a blocker.